src/media/internet/animalviewadoptable/body.html

```diff
@@ -1,6 +1,6 @@
 <div id="asm3-adoptables" />
 <script>
-asm3_adoptable_filters = "sex breed agegroup size species";
+asm3_adoptable_filters = "sex breed agegroup size species goodwith";
```

Comment:
this is making the goodwith filter on by default for all new databases, on reflection, this is probably ok